### PR TITLE
use correct namespace in cuio code examples

### PR DESCRIPTION
**Pull request by cwharris**
use correct namespace in cuio reader/writer code examples

### CODE REVIEW

File: cpp/include/cudf/io/orc.hpp [hunk 1]
```diff
@@ -346,12 +346,9 @@ class orc_reader_options_builder {
  *
  * The following code snippet demonstrates how to read a dataset from a file:
  * @code
- *  ...
- *  std::string filepath = "dataset.orc";
- *  cudf::orc_reader_options options =
- * cudf::orc_reader_options::builder(cudf::source_info(filepath));
- *  ...
- *  auto result = cudf::read_orc(options);
```
Comment (karthikeyann) [inline cpp/include/cudf/io/orc.hpp:354]: other documentation string in this file still has `cudf::orc` usage. 
`cudf::write_`, `cudf::read_` also. Check all the edited files in this PR.

**Review by karthikeyann — CHANGES_REQUESTED** [review]
check all files for similar change in this PR.

**Review by ttnghia — APPROVED** [review]
Please add PR description :smiley:

**Comment by cwharris** [comment]
@ttnghia I was under the impression the title of the PR becomes part of the description...

**Review by karthikeyann — APPROVED** [review]

**Review by devavret — APPROVED** [review]

**Comment by cwharris** [comment]
@gpucibot merge

**Pull request merged**